index.js

```diff
@@ -6,14 +6,15 @@ const { join } = require('path'),
   { readdirSync } = require('fs');
 
 module.exports = {
-  plugins: ['jsdoc', 'lodash', 'mocha', 'node', 'security', 'sonarjs'],
+  plugins: ['jsdoc', 'lodash', 'mocha', 'node', 'security', 'sonarjs', '@typescript-eslint'],
   env: {
     browser: false,
     node: true,
     es6: true
   },
   parserOptions: {
-    ecmaVersion: 2020
+    ecmaVersion: 2020,
+    sourceType: 'module'
```

Comment:
@kunagpal updated this since it complained about `Parsing error: 'import' and 'export' may appear only with 'sourceType: module'` in JS files in Bifrost API. Any concerns?

Reply:
As long as the linting works correctly, we're good.